configure.py

```diff
@@ -449,8 +482,8 @@ def shell_escape(str):
             variables=[('varname', 'kBrowsePy')])
     n.newline()
 
-    objs += cxx('browse', order_only=built('browse_py.h'))
-    n.newline()
+    srcs += [cxxsrc('browse')]
+    order_only_deps['browse'] = built('browse_py.h')
```

Comment:
Possibly, but if we always use the amalgamation to bootstrap, it seems simpler to not special-case the Python support.